install/resources/mybb_theme.xml

```diff
@@ -4206,7 +4206,7 @@ $(".forumjump").on('change', function() {
 </div>]]></template>
 		<template name="global_remote_avatar_notice" version="1809"><![CDATA[<div class="red_alert"><a href="{$mybb->settings['bburl']}/usercp.php?action=avatar">{$lang->remote_avatar_disabled_default_avatar}</a></div>]]></template>
 		<template name="global_unreadreports" version="1820"><![CDATA[<a href="{$mybb->settings['bburl']}/modcp.php?action=reports">{$lang->unread_reports}</a>]]></template>
-		<template name="global_moderation_notice" version="1800"><![CDATA[<div class="pm_alert">{$moderation_text}</div>]]></template>
```

Comment:
Perhaps the version should now be `1827`? I don't know if these are updated for closed PRs ..

Reply:
Done!